src/content/1.7/modules/creation/module-file-structure.md

```diff
@@ -12,6 +12,7 @@ A module is made of a lot of files, all stored in a folder that bears the same n
 
 {{% notice tip %}}
 Your module can be called anything, as long as it only contains lowercase letters and numbers (`/[a-z0-9]/`).
+Please don't use underscores (_) in your module name, it's not compatible with new translation system.
```

Comment:
```suggestion
Although accepted, we strongly discourage using underscores (`_`) because they don't work with translation domains.
```
Same wording as https://devdocs.prestashop.com/1.7/modules/creation/tutorial/